static/templates/settings/realm-domains-modal.handlebars

```diff
@@ -15,13 +15,19 @@
             <tfoot>
                 <tr id="add-realm-domain-widget">
                     <td><input type="text" class="new-realm-domain" placeholder="acme.com"></input></td>
-                    <td><input type="checkbox" class="new-realm-domain-allow-subdomains"></input></td>
+                    <td>
+                        <label class="checkbox">
+                            <input type="checkbox" class="new-realm-domain-allow-subdomains" />
+                            <span></span>
```

Comment:
what is the purpose of this empty span?

Reply:
the empty span is for styling purposes, it's actually the checkbox that's visible to us. The display for the real input checkbox is set to none. On the rest of the places on the settings page, the checkboxes are styled similarly, that's why I used the same here as well :sweat_smile: 
![screenshot from 2017-12-21 11-03-37](https://user-images.githubusercontent.com/22816171/34241930-b730c940-e63e-11e7-8385-87fa350565e5.png)

Reply:
Ah, I guess we do use that in a lot of places.  It feels strange to me not to attach a class or id to the span, but if that's our convention for now, I don't have a problem.